# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 2adbab502a1e
# Full Hash: 2adbab502a1e690fccd61fb604be820e60d61723
# Author: Kristen Wright <kwright@mozilla.com>
# Date: 2020-09-09 09:39:57
# Regressor Bug: 1642727
# File Overlap Count: 1
# Description:
#   Bug 1642727 - Remove the VarCache machinery r=njn
#   
#   This commit removes Varcache entirely, putting an end to our quest to get rid of VarCache.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79538
# ==============================================================================

diff -r 1aeb7707569f -r 2adbab502a1e modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Tue Sep 08 15:05:06 2020 +0000
+++ b/modules/libpref/init/all.js	Tue Sep 08 17:12:01 2020 +0000
@@ -9,8 +9,8 @@
 // mobile/android/app/mobile.js.
 //
 // NOTE: Not all prefs should be defined in this (or any other) data file.
-// Static prefs, especially VarCache prefs, are defined in StaticPrefList.yaml.
-// Those prefs should *not* appear in this file.
+// Static prefs are defined in StaticPrefList.yaml. Those prefs should *not*
+// appear in this file.
 //
 // For the syntax used by this file, consult the comments at the top of
 // modules/libpref/parser/src/lib.rs.